lib-android-robolectric/src/main/java/com/devinci/lib/robolectric/AttributeSetBuilder.java

```diff
@@ -0,0 +1,38 @@
+package com.devinci.lib.robolectric;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nonnull;
```

Comment:
Why not make it android library project? I doubt it'd be used in any plain Java project. In addition we can add xml's to aar file (may be useful in future).